### DISCUSSION TITLE
Can I export my data?

### DISCUSSION THREAD

**Discussion by andriiPalko** [TelemetryDeck/Viewer#253] [Q&A] [answered]
Hello, can I export my data in CSV (or any other) format?

**Comment by winsmith** [comment, marked as answer]
Thanks for the question! You can't use the UI yet to export your data, but we can export it for you. Please write to info@telemetrydeck.com with your organization ID and wether you'd like CSV or JSON format and we'll get back to you with download links in a few days.
